Ignore the date stamps on gtk icon cache files

Clear Linux swupd does not preserve timestamps as part of the swupd process,
and as a result, we have no good way to ensure that the condition in gtk
that checks icon caches for freshness based on date of directories versus
dates of icon cache files succeeds, and the effect is a disasterous amount
of extra memory use for each GTK app that uses icons.

if the timestamp means nothing, the check for the timestamp is meaningless
as a result and we need to just not look at the timestamp.

--- gtk+-3.22.0/gtk/gtkiconcache.c~	2016-03-26 21:32:35.000000000 +0000
+++ gtk+-3.22.0/gtk/gtkiconcache.c	2017-04-30 17:18:51.000318041 +0000
@@ -117,12 +117,14 @@
   if (fstat (fd, &st) < 0 || st.st_size < 4)
     goto done;
 
+#if 0
   /* Verify cache is uptodate */
   if (st.st_mtime < path_st.st_mtime)
     {
       GTK_NOTE (ICONTHEME, g_message ("icon cache outdated"));
       goto done; 
     }
+#endif
 
   map = g_mapped_file_new (cache_filename, FALSE, NULL);
 
